.bazelrc

```diff
@@ -22,6 +27,38 @@ build --features=layering_check
 # TODO(fzakaria): Make this a toolchain or hermetic somehow
 build --repo_env=CC=clang
 
+# turn on fission https://gcc.gnu.org/wiki/DebugFission
+# this separates the debug information and can improve link times
+build --fission=yes
+
+build:dbg --compilation_mode=dbg
+build:opt --compilation_mode=opt
+
+build:san-common --config=dbg --strip=never --copt=-O0 --copt=-fno-omit-frame-pointer
+
+# Some of this is from "Can I run AddressSanitizer with more aggressive diagnostics enabled?"
+# on https://github.com/google/sanitizers/wiki/AddressSanitizer#faq and some is from
+# https://chromium.googlesource.com/external/github.com/grpc/grpc/+/4e9206f48c91e17f43856b016b12f59dd5118293/tools/bazel.rc
+build:asan --config=san-common
+build:asan --features=asan
+build:asan --copt=-fsanitize-address-use-after-scope
+# We explicitly disable strict_init_order because LLVM is "hostile to init order"
```

Comment:
Consider linking to the LLVM asan config for reference.

Reply:
What do you mean ?
I'm trying to hint that the google3 config has whitelist to avoid LLVM for this check.